### PR TITLE
feat: [CDS-52184]: instance sync changes for GoogleCloudFunctions swim-lane

### PR DESCRIPTION
**Pull request by chetanvekariya-harness**
### Summary
- instance sync changes for GoogleCloudFunctions swim-lane in service page

#### Screenshots
Screenshot 2023-02-02 at 2 50 42 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by chetanvekariya-harness** [comment]
retrigger titlecheck

**Comment by chetanvekariya-harness** [comment]
retrigger jest

**Comment by chetanvekariya-harness** [comment]
retrigger coverage

**Review by vikrant-harness — APPROVED** [review]

**Review by Sagar-Rajput7 — APPROVED** [review]

File: src/modules/75-cd/strings/strings.en.yaml [hunk 1]
```diff
@@ -370,6 +370,12 @@ serviceDashboard:
   containerList: Container List
   elastigroupId: Elastigroup Id
   ec2InstanceId: EC2 Instance Id
+  revision: revision
+  functionName: functionName
+  source: source
+  runTime: runTime
+  updatedTime: updatedTime
+  memorySize: memotrySize
```
Comment (rameshx) [inline src/modules/75-cd/strings/strings.en.yaml:378]: typo
```suggestion
  memorySize: memorySize
```

File: src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancePopover.tsx [hunk 2]
```diff
@@ -38,6 +39,23 @@ import css from '@cd/components/ServiceDetails/ActiveServiceInstances/ActiveServ
 
 export type InstanceData = Record<string, Record<string, InstanceDetailsDTO[]>>
 
+/**
+ * @TODO Use this type from cd-ng service file.
+ * Currently BE is not synced with latest BE develop and hence types issues are getting created.
+ * So, did not update cd-ng services and created local type here
+ */
+type GoogleFunctionInstanceInfoDTO = InstanceInfoDTO & {
```
Comment (Sagar-Rajput7) [inline src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancePopover.tsx:47]: please make sure to remove this afterward when the issues get resolved, as all the other swimlane dtos are maintained and synced under cd-ng services only.

Comment (chetanvekariya-harness) [reply src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancePopover.tsx:47]: Of cource. That's why the @TODO there.

**Review by rameshx — APPROVED** [review]

**Review by Sagar-Rajput7 — APPROVED** [review]

**Comment by chetanvekariya-harness** [comment]
retrigger jest

**Comment by chetanvekariya-harness** [comment]
retrigger jest

**Pull request merged**